### ISSUE TITLE
[PROPOSAL] `run` in absence of yaml file should return error

### ISSUE THREAD

**Issue by ijstokes** [Anaconda-Platform/anaconda-project#70]
Sometimes when I execute `anaconda-project run` it can take a minute or even several minutes to first start up: conda environment creation and all that jazz (possibly triggering downloads, possibly 100s of MB of packages, then untaring, linking, etc).

And then there is the time when I accidentally execute `anaconda-project run` in the wrong directory.  I sit and stare.  And wait.  And think "huh, the environment should already have been created, I wonder why it is doing it again?" (because I'm incorrectly assuming that is what I'm waiting for).  And then it is taking too long, and so I CTRL-C, and it is only then that I realize I've executed the command in the wrong directory and Anaconda Project was doing exactly nothing.

So I'd like to suggest that if there isn't a YAML file for `anaconda-project run` to consume (either from a CLI argument or implicitly from an `anaconda-project.yml` file in the current directory) then it should return an error message and exit.

And I don't think it should create an `anaconda-project.yml` file.  That seems to just be a weird side-effect of running the command when a YAML file isn't available.

**Comment by havocp** [comment]
Yes, I think I agree

**Comment by havocp** [comment]
Doh, couldn't find this so opened #79 , will close it again but it has some details on how to implement.

**Comment by mcg1969** [comment]
Ping. This just threw me off.  I'm 100% in favor of this improvement.

**Comment by mcg1969** [comment]
#88 seems to fix this

**Comment by mcg1969** [comment]
Fixed by #176.
```
$ anaconda-project run --directory nonexistent
Project directory '/Users/mgrant/Repos/anaconda-project/examples/nonexistent' does not exist.
Unable to load the project.
$ mkdir nonexistent
$ anaconda-project run --directory nonexistent
Project file 'anaconda-project.yml' does not exist.
Create file '/Users/mgrant/Repos/anaconda-project/examples/nonexistent/anaconda-project.yml'? n
Project file 'anaconda-project.yml' does not exist.
Unable to load the project.
```
